### ISSUE TITLE
Unexpected token while parsing modules

### ISSUE THREAD

**Issue by eramdam** [felixhageloh/uebersicht#234]
I have a simple `index.js` file for my widget, which has a callback for the `command` option, like this

```js
command: (callback) => {
  const myModule = require('./lib/module.js');
  console.log(myModule);

  callback(null, myModule);
}
```

And I have this in `./lib/module.js`

```js
module.exports = 'foo';
```

This should work but I get the following trace

<details>
```
[Error] /Users/damien/Library/Application Support/Übersicht/widgets/NoeudPapillon.widget/index.js: Error: Line 1: Unexpected token . while parsing file: /Users/damien/Library/Application Support/Übersicht/widgets/NoeudPapillon.widget/lib/applescript.js
	exports (client.js:6:25769)
	(anonymous function) (client.js:1:1430)
	(anonymous function) (client.js:1:1706)
	fire (client.js:3:993)
	fireWith (client.js:3:2098)
	done (client.js:4:20348)
	(anonymous function) (client.js:4:28645)
```
</details>

**Comment by felixhageloh** [comment]
from the stack trace it looks like it is also trying to include the file `lib/applescript.js` which has a syntax error. Is this the full index.js file you are showing here?

**Comment by eramdam** [comment]
I did a gist with the actual files I use https://gist.github.com/eramdam/46d44432f2fe0a5dad5f63876bd8d7ec

**Comment by hawkrives** [comment]
TL;DR: Try renaming `applescript.js` to `applescript.jsx`.

Übersicht automatically bundles all `.js` files in an object literal, to mimic what coffeescript does. As such,

```js
command: 'echo "Hello"'
```

is interpreted as 

```js
{
  command: 'echo "Hello"'
}
```

So your `applescript.js` file is being interpreted as 

```js
{ 
  module.exports = 'foo'; 
}
```

which is invalid JS.

If you want to use `import`/`export` instead of `require`/`module.exports`, you'll need to name it with a `.jsx` extension, such as `applescript.jsx` (you can see the heuristic in [server/src/bundleWidget.js#L48](https://github.com/felixhageloh/uebersicht/blob/aefe9ce51fc2ff654f1bc40fea8601a686ca0e9d/server/src/bundleWidget.js#L48)). `.jsx` files don't need to include any JSX syntax, it's just the current way of identifying the alternate widget syntax which uses named exports (`export const command = "echo"` or `module.exports.command = "echo"`) instead of treating the file as one giant object literal (the `command: echo` style.)

Since the `applescript.js` file looks like it's probably not intended to do any widget rendering, you shouldn't need to worry about any of the other differences between the systems.

`require`/`module.exports` should work in all three syntaxes, I believe, since they're all being passed through browserify before rendering.

**Comment by felixhageloh** [comment]
oh yeah sorry I didn't notice that you were using ES2015 syntax here. For standard widgets you currently have to use either CoffeeScript or ES5 (and there is no CommonJS support for ES5 widgets). However you can use ES2015 with new [new (beta) widget syntax](https://gist.github.com/felixhageloh/94d5dd41cc5ff8a633900ffb28bd195a) as @hawkrives mentioned
